openapi.yaml

```diff
@@ -82,109 +222,252 @@ parameters:
     in: header
     type: string
     required: true
-    description: '"Bearer [token]" where [token] is your OAuth2 access token'
-  pageNumber:
-    name: page[number]
-    in: query
-    type: integer
-    required: false
-    description: Page number of results
-    default: 1
-    minimum: 1
-  pageSize:
-    name: page[size]
-    in: query
-    type: integer
-    required: false
-    description: Number of results to return
-    default: 25
-    maximum: 500
-    minimum: 1
+    description: '"Bearer [token]" where token is your OAuth2 access token'
```

Comment:
It would not be allowed.

Reply:
Again, what does "not allowed" mean? You need to define the proper response with its status code for all possible scenario so that your clients can expect what they will receive by using your API.